LoopKit/Service/RemoteDataService.swift

```diff
@@ -104,11 +104,11 @@ public protocol RemoteDataService: Service {
     func uploadSettingsData(_ stored: [StoredSettings], completion: @escaping (_ result: Result<Bool, Error>) -> Void)
     
     /**
-     Validates a push notification originated from this data service.
+     Fetch a RemoteCommand from a push notification
      - Parameter notification: The push notification dictionary
-     - Returns: Success
+     - Returns: RemoteCommand
      */
-    func validatePushNotificationSource(_ notification: [String: AnyObject]) -> Result<Void, Error>
+    func commandFromPushNotification(_ notification: [String: AnyObject]) async throws -> RemoteCommand
```

Comment:
Nit: The docs say "fetch", but the signature looks like a simple conversion. Would be nice if the caller knows if this is a method that may have network level latencies.